common/viewer.h

```diff
@@ -135,6 +135,8 @@ namespace rs2
         bool paused = false;
         bool metric_system = true;
 
+		uint32_t ground_truth_r = 2500;
```

Comment:
Is it used?

Reply:
yes, to save the last value the user typed in tare calibration.